Shared/App/AppDelegate.swift

```diff
@@ -84,5 +77,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         self.mainCoordinator = MainCoordinator(router: router, deepLink: nil)
         self.mainCoordinator?.start()
     }
-}
 
+    func sceneWillResignActive(_ scene: UIScene) {
+        let badgeNumber = UIApplication.shared.applicationIconBadgeNumber
+        UserDefaults(suiteName: Config.shared.environment.groupId)?.set(badgeNumber, forKey: "badgeNumber")
+    }
```

Comment:
The badge update code was previously in "enterForeground" but it makes more sense to me to have in in "resignActive". That way it updates if the badge number was changed while the app was open.